File: Lab1/t1.cpp
#include <stdio.h>
int main(){
    int first, second;
    printf("Enter the first integer: ");
    scanf("%d", &first);//Scan the first integer
    printf("Enter the second integer: ");
    scanf("%d", &second);//Scan the second integer
    if (first >= 0 && second >= 0){//Situations when the 2 numbers are both positive
        printf("The integer %d and %d are both positive.\n", first, second);
    }
    else{//Situations when 2 numbers are not both positive
        printf("The integer %d and %d are not both positive.\n", first, second);
    }
    return 0;
}

File: Lab1/t1.kt
import java.util.*

fun main(args: Array<String>){
    val a: Int
    val b: Int
    val input = Scanner(System.`in`)
    print("Enter the first integer: ")
    a = input.nextInt()
    print("Enter the second integer: ")
    b = input.nextInt()
    if (a >= 0 && b >= 0)
        println("The integer $a and $b are both positive.")
    else
        println("The integer $a and $b are not both positive.")
}